recipes/kariba/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "Kariba" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antonpannekoek/kariba/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f08b75ebd2ae84eb0f716d28eeb64d2adcae9daf9224a5e8ce06edab1ad6f75e
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - gsl
+    - libopenblas   # [linux]
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx]
+
+test:
+  commands:
+    - ls $(dirname $(which kariba_test))/../include/kariba/*.hpp
+    - ls $(dirname $(which kariba_test))/../lib/libkariba.so   # [linux]
+    - ls $(dirname $(which kariba_test))/../lib/libkariba.dylib   # [osx]
```

Comment:
This existence tests typically just use `$PREFIX`, is there any reason why you need to refer to `kariba_test`? Furthermore, you can use the `SHLIB_EXT` env variable to unify the check between linux and osx, see https://conda-forge.org/docs/maintainer/adding_pkgs/#simple-existence-tests .